eng/Versions.props

```diff
@@ -32,7 +32,7 @@
     <NewtonsoftJsonVersion>10.0.3</NewtonsoftJsonVersion>
     <NuGetVersioningVersion>4.4.0</NuGetVersioningVersion>
     <NuGetVersion>4.4.0</NuGetVersion>
-    <OctokitVersion>0.32.0</OctokitVersion>
+    <OctokitVersion>0.36.0</OctokitVersion>
```

Comment:
I had to update this version because I use new functionality in the Rollout Scorer.

Reply:
As long as the other stuff works, that's fine. :-)